util/src/main/scala/stryker4s/extensions/mutationtypes/MutationTypes.scala

```diff
@@ -28,7 +28,35 @@ trait BooleanSubstitution extends Mutation[Lit.Boolean]
 
 trait LogicalOperator extends Mutation[Term.Name]
 
-trait MethodMutator extends Mutation[Term.Name]
+// TODO: check another mutators (maybe there is a better way to match them), find a common interface
+trait MethodMutator {
+  protected val methodName: String
+  def apply(f: String => Term): Term = f(methodName)
+  def unapply(term: Term): Option[(Term, String => Term)]
+}
+
+trait OneArgMethodMutator extends MethodMutator {
+
+  def unapply(term: Term): Option[(Term, String => Term)] = term match {
+    case Apply(Select(q, Name(`methodName`)), Block(Function(_ :: _ :: _, _) :: Nil) :: Nil) =>
```

Comment:
ApplyInfix can be used to match something like ```a + b```, ```a > b``` and ```a && b```

I can change the way Binary and Logical Operators works in order to move it in the same direction than Methods. (and I can add new mutators too, like + -> -, - -> +, * -> / and / -> *

Reply:
Methods should be able to be matched with the infix notation too, like `list filter filterFunc` -> `list filterNot filterFunc`. 

The reason we don't have certain binary operators (like `+`, `-`, etc) is because there's a higher chance of those mutations causing a compile error (`"foo" + "bar"` -> `"foo" - "bar"` for example) and we have no way to deal with those yet